### PR TITLE
Created the PR-template

### PR DESCRIPTION
**Pull request by jaynarayan-vaishnav**
created and added the necessary info to PR template.

<a href="https://gitpod.io/#https://github.com/krshkodes/username-checker/pull/15"></a>

### CODE REVIEW

**Comment by jaynarayan-vaishnav** [comment]
I have completed both the tasks as you have said pls check if its done correctly @krshkodes

**Review by krshkodes — CHANGES_REQUESTED** [review]
It looks good, but in the issue templates there is no end line, please add an empty line at the end, and this PR is ready to go!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by jaynarayan-vaishnav** [comment]
Okay will complete it today✌🏻

**Comment by jaynarayan-vaishnav** [comment]
hey, @krshkodes I have added the end lines on the ISSUE_TEMPLATE files. but there were too many commits so I force pushed it into my forked repo is it okay?

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      required: false
+
+  - type: checkboxes
+    id: guidelines
+    attributes:
+      label: Code of Conduct
+      options:
+        - label: I agree to follow this project's [Code of Conduct](https://github.com/CeruleanCodersComm/.github/blob/main/CODE_OF_CONDUCT.md)
+          required: true
+
```
Comment (krshkodes) [inline .github/ISSUE_TEMPLATE/bug_report.yml:46]:
```suggestion

```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Discord Community
+    url: https://dsc.gg/ceruleancoders
+    about: Join the Discord Community for more awesome stuff!
+
```
Comment (krshkodes) [inline .github/ISSUE_TEMPLATE/config.yml:6]:
```suggestion

```

File: .github/ISSUE_TEMPLATE/documentation.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      required: false
+
+  - type: checkboxes
+    id: guidelines
+    attributes:
+      label: Code of Conduct
+      options:
+        - label: I agree to follow this project's [Code of Conduct](https://github.com/CeruleanCodersComm/.github/blob/main/CODE_OF_CONDUCT.md)
+          required: true
+
```
Comment (krshkodes) [inline .github/ISSUE_TEMPLATE/documentation.yml:32]:
```suggestion

```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+      required: false
+
+  - type: checkboxes
+    id: guidelines
+    attributes:
+      label: Code of Conduct
+      options:
+        - label: I agree to follow this project's [Code of Conduct](https://github.com/CeruleanCodersComm/.github/blob/main/CODE_OF_CONDUCT.md)
+          required: true
+
```
Comment (krshkodes) [inline .github/ISSUE_TEMPLATE/feature_request.yml:43]:
```suggestion

```

File: .github/ISSUE_TEMPLATE/other.yml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+      required: false
+
+  - type: checkboxes
+    id: guidelines
+    attributes:
+      label: Code of Conduct
+      options:
+        - label: I agree to follow this project's [Code of Conduct](https://github.com/CeruleanCodersComm/.github/blob/main/CODE_OF_CONDUCT.md)
+          required: true
+
```
Comment (krshkodes) [inline .github/ISSUE_TEMPLATE/other.yml:32]:
```suggestion

```

**Review by krshkodes — CHANGES_REQUESTED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by jaynarayan-vaishnav** [comment]
i need to remove the space in the end line right? or is it something else pls explain

**Review by krshkodes — APPROVED** [review]
Yeah, it was the spaces! Thanks for the contribution, It looks good to me, thus merging!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Comment by jaynarayan-vaishnav** [comment]
thanks @krshkodes  if there's anything i can help with i would love to do more 😄

**Pull request merged**